auth-web/src/App.vue

```diff
@@ -142,11 +158,14 @@ export default class App extends Mixins(NextPageMixin) {
           return
         }
       }
-      this.setupNavigationBar()
       if (this.signingIn) {
+        this.setupNavigationBar()
```

Comment:
does it have happen twice?can see the same setUpNavigation() invoked below as well..

Reply:
Yeah it's needed twice unfortunately:

1. For when the account changes
2. For signin when the account has not changed but they are still authenticated

It's because the accountSyncReady event is only emitted when the account in session storage has changed.